### PR TITLE
Slurm upgrade

### PR DESCRIPTION
**Pull request by steven-basart**
To be squashed

### CODE REVIEW

File: playbooks/slurm_upgrade.yml [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+        /data/slurm_rpms/{{slurm_all_packages[6]}} \
+        /data/slurm_rpms/{{slurm_all_packages[7]}} \
+        /data/slurm_rpms/{{slurm_all_packages[8]}} \
+        /data/slurm_rpms/{{slurm_all_packages[9]}} \
+        /data/slurm_rpms/{{slurm_all_packages[10]}}
+      # Needed in case you wish to rerun this file it'll error.
+      ignore_errors: true
+
+- hosts: bastion, slurm_backup
+  tasks:
```
Comment (andriy-safe-ai) [inline playbooks/slurm_upgrade.yml:63]: Add `become: true` below `hosts: ...`. That way you don't have to repeat yourself in each task.

Comment (andriy-safe-ai) [reply playbooks/slurm_upgrade.yml:63]: Do that for each play.

**Review by andriy-safe-ai — CHANGES_REQUESTED** [review]

File: playbooks/slurm_upgrade.yml (hunk 1 above)
Comment (steven-basart) [reply playbooks/slurm_upgrade.yml:62]: destroying hopes and dreams of those becomers you know

Comment (steven-basart) [reply playbooks/slurm_upgrade.yml:62]: it has been done though.

**Review by andriy-safe-ai — CHANGES_REQUESTED** [review]

File: playbooks/slurm_upgrade.yml [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        name: nvidia-container
+      when: enroot|default(true)|bool
+    - include_role: 
+        name: nvidia-enroot
+      when: enroot|default(true)|bool
+    - include_role:
+        name: tuned
+
+- hosts: bastion,slurm_backup,compute,login
+  gather_facts: true
```
[the remark this answers is not in the document]
Comment (steven-basart) [reply playbooks/slurm_upgrade.yml:102]: No this calls Oracle's packages which might mess things up if run as root.

**Review by andriy-safe-ai — APPROVED** [review]
LGTM!

**Pull request merged**